docs/glossary.md

```diff
@@ -43,10 +48,10 @@ To version data means creating a unique reference for data collection. This refe
 lakeFS allows teams to treat their data lake as a Git repository. Git-like operations over data are infrastructure, not a feature provided by an application, with data versioning as an essential part of that infrastructure.
 
 ### Graveler
-Graveler is responsible for handling versioning in lakeFS by translating lakeFS addresses to the actual stored objects. See the [data model section](https://docs.lakefs.io/understand/versioning-internals.html) to learn about the data model used to store lakeFS metadata.
+Graveler is responsible for handling versioning in lakeFS by translating lakeFS addresses to the actual stored objects. See the [data model section]({{ site.baseurl }}/understand/versioning-internals.html) to learn about the data model used to store lakeFS metadata.
 
 ### Hooks
-lakeFS hooks allow you to automate and ensure that a given set of checks and validations happens before important lifecycle events. They are similar conceptually to [Git Hooks](https://git-scm.com/docs/githooks), but in contrast, they run remotely on a server. Currently, lakeFS allows executing hooks when two types of events occur: pre-commit events that run before a commit is acknowledged and pre-merge events that trigger right before a merge operation. 
+lakeFS hooks allow you to automate and ensure that a given set of checks and validations happens before important lifecycle events. They are similar conceptually to [Git Hooks]({{ site.baseurl }}/docs/githooks), but in contrast, they run remotely on a server. Currently, lakeFS allows executing hooks when two types of events occur: pre-commit events that run before a commit is acknowledged and pre-merge events that trigger right before a merge operation. 
```

Comment:
This should actually stay an absolute URL: it goes to a different website!  The new version probably does not work.